solana/README.md

```diff
@@ -103,5 +106,42 @@ In order to run them, the Solana programs must be built and their IDL made avail
 To ensure the SDK has the generated IDL, run the tests with the make command:
 
 ```sh
-make anchor-test
-```
\ No newline at end of file
+make test
+```
+
+### Troubleshooting
+
+<details>
+<summary><code>make: *** No rule to make target `test'.  Stop.</code></summary>
+
+- Ensure `Makefile` has target `test`
+</details>
+
+<details>
+<summary><code>tsx: command not found</code></summary>
+
+- Screenshot:
+  <img src="images/tsx-command-not-found.png" alt="tsx command not found screenshot">
+- Update `Makefile` ([line #29](https://github.com/wormhole-foundation/example-native-token-transfers/blob/main/solana/Makefile#L29)) from:
+
+  ```sh
+  tsx scripts/regenerateIdl.ts $$jsonfile > $$tsfile; \
+  ```
+
+  to:
+
+  ```sh
+  npx tsx scripts/regenerateIdl.ts $$jsonfile > $$tsfile; \
+  ```
```

Comment:
Should we just update the Makefile to use `npx tsx` by default?

Reply:
We could update it but it might need to be tested out in different machines. Just `tsx` didn't work for me and so had to update it to `npx tsx` to make it work. There could be cases where `npx` is not globally installed either. 
So I think we can just have it in the troubleshooting section for now and if others run into this, we could accordingly update the Makefile.